server/cmwell-ws/src/pack/application.conf

```diff
@@ -105,7 +105,7 @@ cmwell {
       }
     }
 
-    switch-over-to-zstore.file-size = 512K
+    switch-over-to-zstore.file-size = 256K
```

Comment:
add comment: We have another one like that somewhere else

Reply:
done